test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt

```diff
@@ -20,7 +21,6 @@ timezone --utc UTC
 
 
 
-
 bootloader --location=mbr --append="nofb quiet splash=quiet" 
 
 zerombr\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=100 --asprimary\npart /     --f   stype ext3 --size=1024 --grow\npart swap  --recommended
```

Comment:
These newlines were already there, but that doesn't look normal. Snap issue or real issue?